Live-Snap/Live-Snap/Controllers/CropWallpaperViewController.swift

```diff
@@ -13,13 +13,14 @@ class CropWallpaperViewController: UIViewController, CropViewControllerDelegate
     
     var imageToCrop: UIImage! = UIImage()
     var toolbarInstructionsLabel: UILabel!
+    var cropViewController: CropViewController!
```

Comment:
this is not a good idea as it can cause circular dependencies where cropviewcontroller has a reference to this view controller and vice versa